cassandra/cassandra-full-experiment.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: redskyops.dev/v1beta1
+kind: Experiment
+metadata:
+  name: cassandra-rwx 
+spec:
+  optimization:
+  - name: "experimentBudget"
+    value: "120" #number of trials 
+  parameters:
+  - name: memory
+    min: 500
+    max: 12000
+  - name: cpu
+    min: 500
+    max: 3000
+  - name: MAX_HEAP_SIZE
+    min: 1000
+    max: 8000
+#  - name: HEAP_NEWSIZE
+#    min: 1000
+#    max: 8000
+  constraints:
+    - name: heap_memory
+      sum:
+        bound: "-1500"
+        isUpperBound: false
+        parameters:
+        - name: memory
+          weight: "-1.0"
+        - name: MAX_HEAP_SIZE
+          weight: "1.0"
+#  - order:
+#      lowerParameter: MAX_HEAP_SIZE
+#      upperParameter: memory
+  metrics:
+  - name: duration
+    minimize: true
+    query: "{{duration .StartTime .CompletionTime}}"
+  - name: cost
+    minimize: true
+    query: "{{div (add (mul .Values.cpu 22) (mul .Values.memory 3)) 1000}}"
+  patches:
+  - targetRef:
+      kind: StatefulSet
+      apiVersion: apps/v1
+      name: cassandra
+    patch: |
+      spec:
+        template:
+          spec:
+            containers:
+            - name: cassandra 
+              resources:
+                limits:
+                  cpu: "{{ .Values.cpu }}m"
+                  memory: "{{ .Values.memory }}Mi"
+                requests:
+                  cpu: "{{ .Values.cpu }}m"
+                  memory: "{{ .Values.memory }}Mi"
+              env:
+                - name: MAX_HEAP_SIZE
+                  value: "{{ .Values.MAX_HEAP_SIZE }}M"
+#                - name: HEAP_NEW_SIZE
+#                  value: "{{ .Values.HEAP_NEWSIZE }}M"
+  template: # trial
+    spec:
+      initialDelaySeconds: 15
+      template: # job
+        spec:
+          template: # pod
+            spec:
+              containers:
+              - image: thecrudge/cstress:latest
```

Comment:
This needs to be `ghcr.io/thestormforge/examples-cassandra`, to get the image built [see this example](https://github.com/thestormforge/examples/blob/54e819b7f8a5025fe6f150518efcb3214d63c2a6/.github/workflows/jvm.yaml).